resources/home/dnanexus/utils/query.py

```
@@ -39,8 +41,9 @@ def get_multiqc_report(path_to_reports, project):
     return multiqc
 
 
-def make_url(file_id, project, url_duration):
-    """Given a file id create a download url
+def make_url(file_id, project, url_duration) -> str:
```

Ruff (0.8.0)

Missing type annotation for function argument `file_id`

(ANN001)